tests/test_core.py

```diff
@@ -58,6 +48,16 @@ def test_003_init_in_branch(
     )
 
 
+def test_004_init_with_empty_repo(
+    data_dir: pathlib.Path, empty_repo: pygit2.Repository
+) -> None:
+    core.init()
+
+    assert config.Config.load(path=pathlib.Path('wok.yml')) == config.Config.load(
+        path=data_dir / '002_wok.yml'
```

Comment:
`004_wok.yml`?

Reply:
The idea is to copy the sample file and keep it as it is for this test only.

Reply:
Sure! Addressed in 38cab18000ba58edae6b4a8e4e0554ee9afcb363.